package.json

```diff
@@ -38,17 +38,17 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
+    "@types/fast-json-stable-stringify": "^2.0.0",
```

Comment:
Looks like a bunch of the `@types` modules were moved to `dependencies`. Guessing that was an accident. Can they be move back to `devDependencies`.

Reply:
Done.